spec/abstract-jobs.html

```diff
@@ -199,6 +199,6 @@ <h1> CleanupFinalizationGroup ( _finalizationGroup_ [ , _callback_ ] ) </h1>
       1. If _result_ is an abrupt completion, return _result_.
       1. Else return NormalCompletion(*undefined*).
     </emu-alg>
-    <emu-note type=editor>When called from DoAgentFinalization, if calling the callback throws an error, this will be caught within the RunJobs algorithm and reported to the host. HTML does not apply the RunJobs algorithm, but will also <a href="https://html.spec.whatwg.org/#report-the-error">report the error</a>, which may call `window.onerror`.</emu-note>
+    <emu-note type=editor>When called from HostCleanupFinalizationGroup, if calling the callback throws an error, this will be caught within the RunJobs algorithm and reported to the host. HTML does not apply the RunJobs algorithm, but will also <a href="https://html.spec.whatwg.org/#report-the-error">report the error</a>, which may call `window.onerror`.</emu-note>
```

Comment:
This isn't quite right since `HostCleanupFinalizationGroup` only schedules a call to `CleanupFinalizationGroup` later and doesn't call it directly.

Maybe the note should be moved to the `HostCleanupFinalizationGroup` section above and say something like:
> If the result of calling CleanupFinalizationGroup(_fg_) is an abrupt completion, the host should report the error.

I don't know how to mention the `RunJobs` algorithm there, since again, the host may not add a job to the queue with the way `HostCleanupFinalizationGroup` is formulated.

Any input?

Reply:
> the host should report the error.

That formulation is also probably not right since we can't mandate that the host do report this specific error if it doesn't report other run-time errors.

Maybe I'm just over-thinking this.

Reply:
My personal opinion is that we should remove the RunJobs algorithm, since it doesn't integrate well with host specifications. This is a separate proposal that I have on the agenda for the June 2019 TC39 meeting (text by @domenic).